Sources/GRPCCore/GRPCServer.swift

```diff
@@ -74,7 +74,7 @@ public struct GRPCServer: Sendable {
   typealias Stream = RPCStream<ServerTransport.Inbound, ServerTransport.Outbound>
 
   /// The ``ServerTransport`` implementation that the server uses to listen for new requests.
-  private let transport: any ServerTransport
+  public let transport: any ServerTransport
```

Comment:
Does it make sense to have this `public`? Is there any reason why a user would want to reach out to the transport directly after it's been configured? Otherwise it could just be `package` to be used in the `ListeningServerTransport` extension.

Reply:
The rationale for making it public was that the user passed in the transport, so providing a `public` reference to it doesn't give them anything they didn't already have.